### PR TITLE
* Exposes log file name in exception message

### PR DESCRIPTION
**Pull request by jayzeng**
Display log file name in exception message to help debugging.

### CODE REVIEW

File: lib/IDS/Log/FileLogger.php [hunk 1]
```diff
@@ -203,12 +204,12 @@ public function execute(Report $data)
...
                 }
-            } else {
-                throw new \Exception(
-                    'Given file does not exist. Please make sure the
-                    logfile is present in the given directory.'
-                );
             }
+
+            throw new \InvalidArgumentException(
+                    sprintf('Given file %s does not exist. Please make sure the
```
Comment (lstrojny) [inline lib/IDS/Log/FileLogger.php]: Two things:
- The indentation for sprintf doesn’t look right (4 chars)
- The string should be moved to the next line and indented

File: lib/IDS/Log/FileLogger.php [hunk 2]
```diff
@@ -203,12 +204,12 @@ public function execute(Report $data)
...
+                        logfile is present in the given directory.', $this->logfile)
```
Comment (lstrojny) [inline lib/IDS/Log/FileLogger.php]: The message string should be in one line, the logfile parameter in the next line and the closing brace in another line :)

**Comment by lstrojny** [comment]
Looks good, except the few cosmetic remarks.

File: lib/IDS/Log/FileLogger.php (hunk 2 above)
Comment (jayzeng) [inline lib/IDS/Log/FileLogger.php]: sure thing, not entirely sure if there is any CS I should follow.

**Comment by jayzeng** [comment]
@lstrojny updated CS

**Pull request merged**